formatPkg.js

```diff
@@ -5,6 +5,7 @@ import numeral from 'numeral';
 const defaultGravatar = 'https://www.gravatar.com/avatar/';
 import escape from 'escape-html';
 import traverse from 'traverse';
+import truncate from 'truncate-utf8-bytes';
```

Comment:
Not really understanding why they loop through the whole content? Might be a problem for us?

https://github.com/parshap/truncate-utf8-bytes/blob/master/lib/truncate.js#L26-L39